_metadata/repos.conf.d/epel/provider.conf

```diff
@@ -1,17 +0,0 @@
-4/
-4AS/
-4ES/
-4WS/
-5/
-5Client/
-5Server/
-6Server/
-7Server/
-epel/
-testing/
-*ppc64/
```

Comment:
If we are grabbing the "Everything" directories, I do not think we need the "Server" ones. They include mostly duplicate packages. Also, we should continue to exclude PowerPC. I also still do not see the need to grab testing, delta RPMs, and debug packages.

Reply:
For EPEL I would not get anything older than version 6